traits/MultiDomainTrait.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace intermundia\yiicms\traits;
+
+use yii\base\InvalidConfigException;
+
+trait MultiDomainTrait
+{
+    public function getWebsiteDomains($websiteKey)
+    {
+        return array_merge($this->getFrontendDomains($websiteKey), $this->getBackendDomains($websiteKey));
+    }
+
+    public function getFrontendDomains($websiteKey)
+    {
+        $domains = \Yii::$app->multiSiteCore->websites[$websiteKey]['domains'];
+        if (!array_key_exists('frontend', $domains)) {
+            throw new InvalidConfigException('Multisite config "domains" array must contain sub-array with "frontend" key');
+        }
+        return $domains['frontend'];
+    }
+
+    public function getBackendDomains($websiteKey)
+    {
+        $domains = \Yii::$app->multiSiteCore->websites[$websiteKey]['domains'];
+        if (!array_key_exists('frontend', $domains)) {
```

Comment:
@mirushaki It should be `backend` not `frontend`

Reply:
@thecodeholic this typo is corrected in commit:
https://github.com/intermundia-de/yiicms/pull/28/commits/8e24c6e2d0ed01549c14739465b6486bde58036e
on this branch